content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -8,30 +8,108 @@ due to theft, damages, human errors, or other factors. As such, inventory adjust
 to reconcile the differences, and ensure that the recorded counts in the database match the actual
```

Comment:
Very minor, but I would put theft at the end of the list, since it's the most negative : due to damages, human errors, theft, or other factors. Just feels "nicer"

Reply:
Makes sense to me!